src/main/resources/application.conf

```diff
@@ -1,6 +1,7 @@
 ergo {
   # Directory to keep data
   directory = ${user.dir}"/ergo/data"
+  directory = ${?DATADIR}
```

Comment:
Not sure if previous `directory = ${user.dir}"/ergo/data"` overwrited to empty string if DATADIR not set